.github/workflows/commands.yml

```diff
@@ -80,13 +80,7 @@ jobs:
         permission-level: write
 
     - name: Checkout
-      uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
+      uses: actions/checkout@v3
 
     - name: Open Backport PR
-      uses: zeebe-io/backport-action@v0.0.4
-      with:
-        github_token: ${{ secrets.GITHUB_TOKEN }}
-        github_workspace: ${{ github.workspace }}
-        version: v0.0.4
+      uses: zeebe-io/backport-action@v1
```

Comment:
do you want to replace this one as well?

Reply:
Good find! Yes I'll adjust it immediately